# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bochs/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual gcc >= 3.4.x cought type mismatch problems.

  - Rene Rebe <rene@exactcode.de>

--- ./fpu/reg_ld_str.c.vanilla	2004-11-15 23:13:58.934640526 +0100
+++ ./fpu/reg_ld_str.c	2004-11-15 23:14:03.933843147 +0100
@@ -315,7 +315,7 @@
 
 
 /* Get a short from user memory */
-int  BX_CPP_AttrRegparmN(1)
+int  BX_CPP_AttrRegparmN(2)
 FPU_load_int16(bx_address _s, FPU_REG *loaded_data)
 {
   s16 s, negative;
